src/main/java/com/comcast/zucchini/ZucchiniShutdownHook.java

```diff
@@ -117,10 +120,24 @@ public void run() {
                     rptName = System.getenv(NAME_ENV_VAR);
 
 
-                ReportBuilder reportBuilder = new ReportBuilder(pathList, html, "", rptName + version, "Zucchini" + version, true, true, true, false, false, "", false);
+                ReportBuilder reportBuilder = new ReportBuilder(pathList, html, "", rptName + version, "Zucchini" + version, 
+                        true, true, true, false, false, "", false);
                 reportBuilder.generateReports();
-
                 boolean buildResult = reportBuilder.getBuildStatus();
+                
+                if ((options != null) && (options.failureReport())) {
```

Comment:
Please move this to a separate method. This will be easier to read and will be more clear to to understand what this is responsible for.